src/index.html

```diff
@@ -3,11 +3,311 @@
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700;800&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
   <body>
-    <h1>Miami</h1>
+    <header id="home" class="header">
+      <a href="#home" class="header__logo">
+        <img
+          class="header__logo-image"
+          src="images/logo.png"
+          alt="Bose logo"
+        >
+      </a>
+
+      <div class="header__action">
+        <a class="header__phone phone" href="tel:+1 234 555-55-55">
+          <img
+            class="phone__icon"
+            src="images/phone.svg"
+            alt="Phone icon"
+          >
+        </a>
+
+        <span class="phone__number">+1 234 555-55-55</span>
+
+        <a class="header__menu" href="#menu">
+          <img
+            class="header__menu-icon"
+            src="images/menu.svg"
+            alt="Menu icon"
+          >
+        </a>
+      </div>
+
+      <div class="header__content">
+        <h1 class="header__motto">
+          The world shades.
+          <br>
+          Your music shines.
+        </h1>
+      </div>
+    </header>
+
+    <nav id="menu" class="menu">
+      <a href="#home">
+        <img
+          class="header__logo"
+          src="images/logo.png"
+          alt="Bose logo"
+        >
+      </a>
+
+      <a class="menu__close" href="#header">
+        <img
+          src="images/cross.svg"
+          alt="Menu close"
+        >
+      </a>
+
+      <ul class="menu__list">
+        <li class="menu__item">
+          <a class="menu__link" href="#home">Home</a>
+        </li>
+
+        <li class="menu__item">
+          <a class="menu__link" href="#recommended">recommended</a>
+        </li>
+
+        <li class="menu__item">
+          <a class="menu__link" href="#categories">categories</a>
+        </li>
+
+        <li class="menu__item">
+          <a class="menu__link" href="#buy-reasons">How to buy</a>
+        </li>
+
+        <li class="menu__item">
+          <a class="menu__link" href="#contact">Contacts</a>
+        </li>
+      </ul>
+
+      <p class="menu__number">+1 234 5555-55-55</p>
+
+      <a class="menu__call" href="tel:+1 234 555-55-55">call to order</a>
+    </nav>
+
+    <main>
+      <section id="recommended" class="recommended">
+        <h1 class="recommended__title">Recommended</h1>
+
+        <div class="recommended__soundbars">
+          <img
+            class="recommended__picture"
+            src="images/photos/soundbars/smart/smart-320.png"
+            alt="Bose portable Smart speaker"
+          >
+
+          <div class="recommended__details">
+            <h3 class="recommended__name">Bose portable Smart speaker</h3>
+
+            <p class="recommended__kind">Smart home</p>
+
+            <h3 class="recommended__price">$ 399.00</h3>
```

Comment:
Avoid using 'h3' for the price. Use a 'p' tag with a class instead.

Reply:
I set tag h3.